### DISCUSSION TITLE
Github Student Pack

### DISCUSSION THREAD

**Discussion by ghazi23-dev** [community/community#140734] [GitHub Education]
### Hi everyone,

Hi everyone, i'm having some issues to activate the github student pack my membership already expired each time i try to reapply for new one i get Hi, Ghazi  ! You were last verified as a student on . It is not necessary for you to re verify at this time. its been like 3 days trying to fix the issue or bypass the error but without any solution.

**Comment by sigfriedherzog001** [comment]
Same problem here.  I was verified as a student but the license expires in two days for me and I can't renew the license, even though my profile shows I'm approved as a student.  I didn't get an email with the license or instructions.  Help!

**Reply by Soodharshal** [reply]
same with me, I am verified , but not able to authorize the IntelliJ with GitHub,

**Reply by ghazi23-dev** [reply]
did you guys find a solution for this issue ? now its says its expired but i can't renew the pack as its displaying the same issue

**Comment by Soodharshal** [comment]
Thank you for your response. But no luck even reach out jet brain but no
solution found yet

**Reply by ghazi23-dev** [reply]
is there a way we can reach out the support of github or open a ticket ?

**Comment by Soodharshal** [comment]
I am looking for a same , but don't know how to reach out to them, direct
mail to GitHub support doesn't work as well.

**Comment by sigfriedherzog001** [comment]
To recap my problem:
My JetBrians account is linked to my GitHub account. However, when I try to get the Jet Brians license, JetBrains will redirect me to GitHub and then GitHub redirects me to the URL https://www.jetbrains.com/community/education/?authMethod=github#students

So, I'm stuck in a endless cycle between jetbrains website and github's website with no ability to apply the student license.

I was able to confirm I was approved with a student license.  In my GitHub account.  Here what can be done.

Step 1) validate you are approved for the student pack for Jet Brians at: https://education.github.com/discount_requests/application?type=student

If approved you will see a request status to the right side and it will show a purple pill type icon with "Approved".

Step 2.1) If you are not approved then open a support ticket with Github to ask about the your request.

Step 2.1) If you are approved, then check if you have the "blue pill" icon that is described with "AUTHORIZE WITH GITHUB" at:
https://www.jetbrains.com/shop/eform/students
See: "Apply with"
click: Github menu item, to view 
"AUTHORIZE WITH GITHUB".  Once I clicked this it applied my new student license with Jet Brains. Of course you'll be required to log into your GitHub account that is linked with Jet Brains to view this option.  Although my account was linked with GitHub already, I had to click the "AUTHORIZED WITH GITHUB" after confirming my new license.

To open a support ticket with GitHub, go to Contact us at:
https://support.github.com/contact/education
Complete the 'Get help with GitHub Education' form to open a support ticket with GitHub.

To open a support ticket with JetBrians go to:
https://education.github.com/pack
Scroll down to "All offers", then see "JETBRAINS"
click the link for "Get help at: JetBrains support"
Click on JetBrains product and then click on the "Contact Support"
